corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -200,27 +200,21 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
         return view;
     };
 
+    var buildCellWrapperStyle = function () {
+        var outerGridTemplateString = $("#cell-wrapper-style-template").html(),
```

Comment:
nitpick: you could just return this line, right, since there's no logic or data in the template?

Reply:
Good point -- unless you have strong feelings against it, I think I'd prefer to leave it as a template so it still lives next to the other styles that apply to case tiles. But for sure no need to have a function that's not doing anything.